pymc3/model.py

```diff
@@ -218,34 +229,39 @@ def cont_vars(self):
         return list(typefilter(self.vars, continuous_types))
 
     def Var(self, name, dist, data=None):
-        """Create and add (un)observed random variable to the model with an appropriate prior distribution.
+        """Create and add (un)observed random variable to the model with an
+        appropriate prior distribution.
 
         Parameters
         ----------
-            name : str
-            dist : distribution for the random variable
-            data : arraylike (optional)
-               if data is provided, the variable is observed. If None, the variable is unobserved.
+        name : str
+        dist : distribution for the random variable
+        data : array_like (optional)
+           If data is provided, the variable is observed. If None,
+           the variable is unobserved.
+
         Returns
         -------
-            FreeRV or ObservedRV
+        FreeRV or ObservedRV
         """
         if data is None:
             if getattr(dist, "transform", None) is None:
                 var = FreeRV(name=name, distribution=dist, model=self)
                 self.free_RVs.append(var)
             else:
-                var = TransformedRV(name=name, distribution=dist, model=self, transform=dist.transform)
+                var = TransformedRV(name=name, distribution=dist, model=self,
+                                    transform=disT.transform)
                 if self.verbose:
                     print('Applied {transform}-transform to {name}'
                           ' and added transformed {orig_name} to model.'.format(
-                              transform=dist.transform.name,
+                              transform=disT.transform.name,
```

Comment:
Bah, sorry. I though I should double check after doing the replace and completely forgot about it :)